libs/s25main/Loader.cpp

```diff
@@ -264,6 +269,25 @@ bool Loader::LoadSounds()
     return true;
 }
 
+bool Loader::LoadFonts()
+{
+    if(!LoadFile(RTTRCONFIG.ExpandPath("<RTTR_RTTR>/LSTS/OUTLINE_FONTS.LST"), GetPaletteN("pal5")))
+        return false;
+    fonts.clear();
+    const auto& loadedFonts = GetArchive("outline_fonts");
+    for(int i = 0; i < 3; i++)
```

Comment:
we should make sure that the archive contains 3 fonts here

Reply:
It does: See 3 lines below

Reply:
ah okay. I thought about a good way to not hardcode the "3" here - perhaps at least add some kind of constant `FONT_COUNT` or similar

Reply:
I added an enum `FontSize` and changed the signature of `GetFont` accordingly. This also makes sure you can only get valid fonts.